proto/nftmarket/nftmarket.proto

```diff
@@ -43,18 +43,14 @@ message ListedNft {
 }
 
 message NftListing {
-  NftIdentifier nft_id       = 1 [(gogoproto.nullable) = false];
-  string        owner        = 2;
-  ListingType   listing_type = 3;
-  ListingState  state        = 4;
-  string        bid_token    = 5;
-  string        min_bid      = 6 [
-    (gogoproto.moretags)   = "yaml:\"min_bid\"",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable)   = false
-  ];
-  uint64                    bid_active_rank = 7;
-  google.protobuf.Timestamp started_at      = 8
+  NftIdentifier             nft_id               = 1 [(gogoproto.nullable) = false];
+  string                    owner                = 2;
+  ListingType               listing_type         = 3;
+  ListingState              state                = 4;
+  string                    bid_token            = 5;
+  uint64                    minimum_deposit_rate = 6;
```

Comment:
What's the unit of deposit rate?

Reply:
This makes little sense.
Always lister would enter the lowest deposit interest rate the lowest.
According to Kimura, advanced users need it.

If a bid is submitted with a bid deposit below the minimum interest rate, the bid will be rejected

Reply:
I mean if deposit rate is in percentage value or if the unit is basis point (1/10000)